cypress/tests/CreateModels.js

```diff
@@ -52,6 +53,7 @@ export function WaitVsNoWaitActions()
   actions.CreateNewAction({ response: 'Cows say moo!', uncheckWaitForResponse: true })
   actions.CreateNewAction({ response: 'Ducks say quack!', uncheckWaitForResponse: true })
   actions.CreateNewAction({ response: 'Fish just swim.', uncheckWaitForResponse: true })
+  actions.CreateNewAction({ type: 'END_SESSION', response: "That's All Folks.", uncheckWaitForResponse: true })
```

Comment:
I didn't think this should be possible.  I would expect it to fail when given those parameters.

Expecting: `uncheckWaitForResponse: false`

Reply:
See prior answer.